[CIAPP] Add "Ignore new flaky tests" section in Flaky Tests management for CI Visibility (#13284)
DataDog/documentation: content/en/continuous_integration/guides/flaky_test_management.md

3. Look at the **New Flaky** column to see the number of new flaky tests that were introduced by the latest commit. These are tests that exhibit flaky behavior and didn’t previously exist in the Flaky Tests table.

### Ignore new flaky tests detected by mistake

> You can ignore new flaky tests detected for a certain commit if you consider those flaky tests were detected by mistake. The tests will reappear if the commit exhibits Flakiness again.

Reviewer: ```suggestion
You can ignore new flaky tests for a particular commit if you determine that those flaky tests were detected by mistake. The tests reappear if the commit exhibits flakiness again.
```

> 1. Click on the **New Flaky** number and then click on the **Ignore flaky tests** action.

Reviewer: ```suggestion
Click on the **New Flaky** number and then click **Ignore flaky tests**.
```
